toolkit/tools/pkg/imagecustomizerlib/customizepackages_test.go

```diff
@@ -211,3 +214,120 @@ func TestCustomizeImagePackagesDiskSpace(t *testing.T) {
 	assert.ErrorContains(t, err, "failed to customize raw image")
 	assert.ErrorContains(t, err, "failed to install package (gcc)")
 }
+
+func ensureTdnfCacheCleanup(t *testing.T, imageConnection *ImageConnection, dirPath string) error {
+	// Get version of base image
+	osVersion := getImageVersion(imageConnection)
+
+	// Variables to capture root directory contents
+	var rootDirContents []string
```

Comment:
I think the `ensureTdnfCacheCleanup` function could be greatly simplified by doing this instead:

1. Create a list of sub-directories in `/var/cache/tdnf` and the number of files within each sub-directory.
2. Verify all sub-directories are empty, except for `local-repo` on `2.0`.